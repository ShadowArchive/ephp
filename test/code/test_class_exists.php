<?php

print "class Iterator => ";
var_dump(class_exists("Iterator"));

print "class NoExist => ";
var_dump(class_exists("NoExist"));

print "class stdClass => ";
var_dump(class_exists("stdClass"));

