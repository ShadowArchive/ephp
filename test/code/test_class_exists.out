class Iterator => bool(false)
class NoExist => bool(false)
class stdClass => bool(true)
